.github/workflows/test-build-server.yml

```diff
@@ -1,20 +1,44 @@
-name: Build Server on Pull Request
+name: Test Server on Pull Request
 
 on:
   pull_request:
 
-env:
-  IMAGE_NAME: citrineos-server
-
 jobs:
   build:
-    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        os: [ubuntu-latest]
+    runs-on: ${{ matrix.os }}
 
     steps:
       - uses: actions/checkout@v4
 
       - name: Set up Docker Buildx
         uses: docker/setup-buildx-action@v3
 
-      - name: Build image
-        run: docker build . --file ./Server/deploy.Dockerfile --tag $IMAGE_NAME
+      - name: Set up QEMU
+        uses: docker/setup-qemu-action@v1
+
+      - name: Set permissions on the uploads directory
+        run: |
+          sudo chown 1000:1000 ${{ github.workspace }}/Server/data/directus/uploads
+
+      - name: Build and Run Docker Compose
+        run: docker-compose -f ${{ github.workspace }}/Server/docker-compose.yml up -d
```

Comment:
This should be 'docker compose' not 'docker-compose', [see docker docs](https://docs.docker.com/compose/migrate/#what-are-the-differences-between-compose-v1-and-compose-v2).

Reply:
Change commited.